src/test/java/org/mifos/integrationtest/cucumber/stepdef/ValidateChannelDef.java

```diff
@@ -0,0 +1,296 @@
+package org.mifos.integrationtest.cucumber.stepdef;
+
+import static com.google.common.truth.Truth.assertThat;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+
+import io.cucumber.core.internal.com.fasterxml.jackson.core.JsonProcessingException;
+import io.cucumber.core.internal.com.fasterxml.jackson.databind.JsonNode;
+import io.cucumber.core.internal.com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+import io.restassured.RestAssured;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+import java.util.ArrayList;
+import java.util.List;
+import org.mifos.connector.common.gsma.dto.CustomData;
+import org.mifos.connector.common.gsma.dto.GsmaTransfer;
+import org.mifos.connector.common.gsma.dto.Party;
+import org.mifos.integrationtest.common.Utils;
+import org.mifos.integrationtest.common.dto.ErrorDetails;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class ValidateChannelDef extends BaseStepDef {
+
+    @Autowired
+    ScenarioScopeState scenarioScopeState;
+
+    Logger logger = LoggerFactory.getLogger(VoucherManagementStepDef.class);
+
+    @Given("I can create a negative TransactionChannelRequestDTO")
+    public void iCanCreateANegativeTransactionChannelRequestDTO() {
+        StringBuilder jsonBuilder = new StringBuilder();
+        jsonBuilder.append("{").append("\"payer\": {").append("\"partyIdInfo\": {").append("\"partyIdType\": \"MSISDN\",")
```

Comment:
Use a DTO for this, this is difficult to maintain

Reply:
done